internal/config/config_integration_test.go

```diff
@@ -22,11 +22,11 @@ func TestConfigSetLogLevel(t *testing.T) {
 
 	// Set the valid log levels
 	for _, l := range []string{
-		"ERROR",
-		"WARN",
-		"INFO",
-		"DEBUG",
-		"TRACE",
+		"Error",
```

Comment:
See the related comment.  We should still allow the user to use case-insensitive values.